openmetrics/assets/configuration/spec.yaml

```
@@ -0,0 +1,12 @@
+name: OpenMetrics
+files:
+  - name: openmetrics.yaml
+    options:
+    - template: init_config
+      options:
+        - template: init_config/default
+        - template: init_config/prometheus
+    - template: instances
+      options:
+        - template: instances/default
+        - template: instances/prometheus
```

```suggestion
        - template: instances/prometheus
        - template: instances/default
```

I think it's more user friendly to keep `prometheus_url` and other specific options on the top.